Composer/packages/client/src/utils/fileUtil.ts

```diff
@@ -6,19 +6,19 @@ import moment from 'moment';
 import { FileTypes, SupportedFileTypes } from '../constants';
 import { File } from '../store/types';
 
-export function getExtension(filename?: string): string | any {
-  if (typeof filename !== 'string') return filename;
-  return filename.substring(filename.lastIndexOf('.') + 1, filename.length) || filename;
+export function getExtension(filename: any): string {
```

Comment:
I don't understand why these changes were made. If we want better type safety we can write this as

```ts
function getExtension<T = string>(filename?: T): T | undefined
```